base/multidimensional.jl

```diff
@@ -95,6 +95,10 @@ module IteratorsMD
     # access to index tuple
     Tuple(index::CartesianIndex) = index.I
 
+    # equality
+    import Base: ==
+    ==(a::CartesianIndex{N}, b::CartesianIndex{N}) where N = a.I == b.I
```

Comment:
No need for the `import`, this can just be `Base.:(==)(...)`, which is more explicit.